docs/pyqgis_developer_cookbook/crs.rst

```diff
@@ -126,7 +126,7 @@ Output:
    Ellipsoid Acronym: EPSG:7030
    Proj String: +proj=longlat +datum=WGS84 +no_defs
    Is geographic: True
-   Map units: DistanceUnit.Degrees
+   Map units: 6
```

Comment:
If you know how we could display the actual unit instead of the enum, it would be great!